src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
Fair enough, but given that command files are non-standardized, it'd still need to dissect the command and check that it's running with e.g. `gcc`, or `clang`, or `ccache XYZ` (wherein there'd need to be a `@` prefix on the command file).

I feel like we've gone beyond the scope of the PR. I'm not against doing something like that in the future, and have included a TODO. That said, the Windows tests are a GA blocker; trading one monkeypatch for another monkeypatch is not.